bundles/org.openhab.binding.kvv/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -8,18 +8,14 @@
 		<description>The KVV bridge connects to the KVV API.</description>
 
 		<config-description>
-			<parameter name="maxTrains" type="integer" required="true" min="0" max="127">
+			<parameter name="maxTrains" type="integer" required="true" min="1" max="127">
```

Comment:
Why not removing the "required" attribute and defining a default value set to 1 which is the default value you consider in the class ?

Reply:
Good point!

Reply:
You have to add `<default>1</default>`

Reply:
I did it.